webrtc.html

```diff
@@ -1873,6 +1999,17 @@
             may be applied as a response to an SDP offer or as an update to a
             previously sent SDP "pranswer".</p>
           </dd>
+
+          <dt>rollback</dt>
+
+          <dd>
+            <p>An RTCSdpType of "rollback" indicates that a description should be
+            treated as an canceling the current SDP negotiation and moving back
+            to the SDP [[!SDP]] offer and answer from the previos stable
+            state. Note the previous stable state could be null if there has
+            not yet been a successful offer-answer negotiation.</p>
```

Comment:
Nit: note -> note that
The previous state can't be null - the localDescription can be null. According the definition, "stable" is the name of the state before negotiation takes place too (RTCPeerState enum definition).